conduct.md

```diff
@@ -8,6 +8,37 @@
 ## Categories of Conduct
 
 * Overall Conduct
```

Comment:
Hi, I'm a member of the markdown team and I've been trying to review some of the pull requests and make sure that the branches adhere to the markdown style in the [markdown repository](https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-markdown) . I noticed that there is a small issue in your branch where you need to change "Overall Conduct" to be a header rather than a list item. Please let me know if you have additional questions about the markdown style. Thanks!

Reply:
Thank you for checking this.